contracts/connection-router/src/state.rs

```diff
@@ -89,7 +89,7 @@ impl<'a> IndexList<ChainEndpoint> for ChainEndpointIndexes<'a> {
 
 #[cw_serde]
 pub struct NewMessage {
-    pub uid: CrossChainUid,
+    pub cc_id: CrossChainId,
```

Comment:
this is a pretty cryptic name for this field, I think this needs a better name. just `id` is fine with me. `uid` is ok. `cc_id` is just too cryptic. This struct is public facing and throughout the entire codebase, so I think it's important to keep it clean.